letter_opener_web.gemspec

```diff
@@ -17,18 +17,12 @@ Gem::Specification.new do |gem|
 
   gem.files         = `git ls-files`.split($INPUT_RECORD_SEPARATOR)
   gem.executables   = gem.files.grep(%r{^exe/}).map { |f| File.basename(f) }
-  gem.test_files    = gem.files.grep(%r{^(test|spec|features)/})
   gem.require_paths = ['lib']
 
   gem.add_dependency 'actionmailer', '>= 5.2'
```

Comment:
Do u know if 5.x still works with the gem? I wouldn't mind bumping the dep to 6.x or 7.x if it gets into our way

Reply:
Not sure. I've been using 6.1 since that's what's in the `Gemfile.lock`.